isso/config.py

```diff
@@ -78,10 +78,28 @@ def getboolean(self, key):
 class IssoParser(ConfigParser):
     """Parse INI-style configuration with some modifications for Isso.
 
-        * parse human-readable timedelta such as "15m" as "15 minutes"
-        * handle indented lines as "lists"
+        * Parse human-readable timedelta such as "15m" as "15 minutes"
+        * Handle indented lines as "lists"
+        * Handle quoted values
+        * Disable string interpolation ('%s') for values
     """
 
+    def __init__(self, *args, **kwargs):
+        """Supply modified defaults to configparser.ConfigParser
+        https://docs.python.org/3.9/library/configparser.html#configparser.ConfigParser
+        allow_no_value=True: Allow empty config keys
+        interpolation=None:  Do not attempt to use python string interpolation on
+                             values. This is especially important for parsing
+                             passwords that might contain '%'
+        """
+        return super(IssoParser, self).__init__(
+            allow_no_value=True, interpolation=None, *args, **kwargs)
+
+    def get(self, section, key, *args, **kwargs):
+        # Remove leading and trailing quotes from values
```

Comment:
Not a fan of this since it's not strictly needed any actually disallows passwords starting or ending with these characters (in addition to whitespace that may actually be gobbled up by the ini parser)

Reply:
+1

Reply:
Alright, thank you for your input! Adjusted accordingly.